spec/formats/configuration_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe "Format configuration" do
+  supertypes = YAML.load_file("app/formats/supertypes.yml")
```

Comment:
Would a 'let' be more conventional here?

Reply:
As it turns out, you can only use `let` inside an `it` block (there are other problems with `let` but we'll talk about those later :)).

Reply:
Clarification: you can only use `let` in a `describe` block, and the resulting variable inside an `it` block. Otherwise it will raise an error like:

> `foo` is not available on an example group (e.g. a `describe` or `context` block). It is only available from within individual examples (e.g. `it` blocks) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc).